src/audit-log/Cargo.toml

```diff
@@ -11,4 +11,5 @@ anyhow = "1.0.57"
 bytesize = { version = "1.1.0", features = ["serde"] }
 serde = { version = "1.0.137", features = ["derive"] }
 serde_json = "1.0.81"
+serde_plain = "1.0.0"
```

Comment:
TIL about this crate. Should it be used in the `serialize` and `deserialize` fns instead of using json as the backing bytes impl? I chose JSON because it seemed to be the only serde impl maintained in the actual serde repo, so I assumed it had the most testing among the serializers. But I'm a bit worried that we might accidentally hit some JSON edge case that's not handled well (numbers come to mind).

Reply:
No no I think `serde_json` was the right call! `serde_plain` _only_ does enum variants, basically. It exists so that you can re-use Serde's support for auto-deriving names based on enum variants. You wouldn't be able to use it to serialize the whole blob.